easybuild/easyconfigs/i/iccifort/iccifort-2016.0.109-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,22 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+easyblock = "Toolchain"
+
+name = 'iccifort'
+version = '2016.0.109'
+
+homepage = 'http://software.intel.com/en-us/intel-cluster-toolkit-compiler/'
+description = """Intel Cluster Toolkit Compiler Edition provides Intel C,C++ and fortran compilers, Intel MPI and Intel MKL"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+gcc = 'GCC'
```

Comment:
as pointed out by @ocaisa in #2214, this should be `GCCcore`, and `binutils` should be included as well as a dep